Loop/main.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  main.swift
+//  Loop
+//
+//  Created by Rick Pasetto on 10/27/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+
+#if FORCE_ENGLISH
+UserDefaults.standard.set(["en"], forKey: "AppleLanguages")
+#else
+UserDefaults.standard.set(nil, forKey: "AppleLanguages")
+#endif
+UserDefaults.standard.synchronize()
+
+UIApplicationMain(
```

Comment:
What does this part of the change do? (changing from synthesized `@UIApplicationMain` to explicit one).

Reply:
`@UIApplicationMain` synthesizes the equivalent of this `main.swift` file.  You have to override the locale even before `UIApplicationMain` is called, which is why I had to create a manual `main.swift`.  As far as I know, there's no way to have any code execute before `UIApplicationMain` if you use `@UIApplicationMain`.